tasks.py

```diff
@@ -24,12 +63,15 @@ def docs(ctx: invoke.Context) -> None:
         "meta",
     ]
 
+    if not sys.platform.lower().startswith("win"):
```

Comment:
The proper check here would be 

`if sys.platform != 'win32'`

Reply:
Sure. I can go with that.